app/models/relationship.rb

```diff
@@ -0,0 +1,7 @@
+class Relationship < ApplicationRecord
+  belongs_to :follower, class_name: "User"
```

Comment:
```suggestion
  belongs_to :follower, class_name: User.name
```
cho hỏi sao phải thêm `class_name ` vào thế cu

Reply:
class_name thêm vào để chỉ định lớp quan hệ khi tên quan hệ khác với tên lớp, ở đây lớp quan hệ với relationship là user. Ở trên nếu không khai báo class_name, rails sẽ hiểu lớp quan hệ là follower